### PR TITLE
new topic for explaining Beta, EA, GA, Deprecated

### PR DESCRIPTION
**Pull request by mystiberry-okta**
Hi @tthompson-okta  or @jpf or @johngronberg-okta 

This new topic uses the api-lifecycle tags. When I try to build locally, this error pops up and the build fails:

Liquid Exception: Liquid syntax error (line 34): Unknown tag 'api' in /home/vagrant/okta/okta.github.io/_source/_docs/api/getting_started/releases-at-okta.md

However, the tags aren't on line 34, rather they are on lines 40, 48, and 64

Is there a problem using the tags outside /api/resources? Something else I've done wrong?

@richardmateosian-okta FYI you don't have to edit this, but if you'd like to, go ahead!

### CODE REVIEW

**Comment by johngronberg-okta** [comment]
try api_lifecycle not api-lifecycle

File: _source/_docs/api/getting_started/releases-at-okta.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+Features of the Okta platform travel through a regular lifecycle:
+
+* [Beta](#beta)
+* [Early Access (EA)](#early-access-ea)
+* [General Availability (GA)](#general-availability-ga)
+* [Deprecation](#deprecation)
+
+This lifecycle applies to most features and behaviors of the platform.
+However, Okta is a shared service, so we reserve the right to change a few
```
Comment (karlmcguinness-okta) [inline _source/_docs/api/getting_started/releases-at-okta.md]: this sounds a bit scary as EA/GA have strict compatibility rules that we are establishing as contract (see http://developer.okta.com/docs/api/getting_started/design_principles.html).  Is there a better way to express this without it sounding like we can break you at will?

File: _source/_docs/api/getting_started/releases-at-okta.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+however, the timeline for addressing specific areas of feedback (including bugs) is determined by Okta's market requirements for that feature.
+Any customer interested in a feature in Beta must apply by visiting [our Beta Signup page](https://oktabeta.zendesk.com/hc/en-us). 
+Only customers with use cases that match our use cases are invited.
+
+Beta releases are either high touch or low touch:
+
+* High-touch Beta releases involve regular contact with Okta, typically consisting of conference calls covering specified use cases, deployment guidance, and feedback.
+* Low-touch Beta releases are self-directed without ongoing support and limited to collecting feedback.
+
+Okta only enables Beta features in non-production or sandbox environments, because features in Beta are not yet supported by Okta Customer Support and
```
Comment (karlmcguinness-okta) [inline _source/_docs/api/getting_started/releases-at-okta.md]: This is the most important concept I think for this section and it feels like it gets lost to a lost of explanation about how we run beta's and not "what" the beta lifecycle contract is for an API.  @johngronberg-okta do we really nee so much text about the low/high touch and how we execute beta programs here?

File: _source/_docs/api/getting_started/releases-at-okta.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+may change at any time during the Beta release.
+
+Features in Beta release are marked with the Beta icon: {% api_lifecycle beta %}
+
+## Early Access (EA)
```
Comment (karlmcguinness-okta) [inline _source/_docs/api/getting_started/releases-at-okta.md]: I think this section lacks some specifics about APIs vs product features. 

- Product features can EA or GA without the API being EA (for example a beta API for a EA feature like System Log 2)
- When an API hits EA the contract will be backwards compatible according to our versioning rules
- APIs might go from beta to GA if they have no product feature flag dependencies
- While we strive for API first development, we also may omit functionality in the EA API that may be added later.

File: _source/_docs/api/getting_started/releases-at-okta.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+Features that have been deprecated are marked with the Deprecated icon: {% api_lifecycle deprecated %}
+
+When Okta has an end-of-life plan, information about the timing will be included with the icon.
+
+## Quick Reference Table
+
+| Description | Beta (High touch) | Beta (Low touch) | EA |  GA  | Deprecated |
+|:------------|:------------------:|:-----------------:|:---:|:---:|:---:|
+| Contact with Product Team  |     X              |                   |       |       |       |
+| Stable API                 |   Subject to change  | Subject to change | X   | X   | X   |
```
Comment (karlmcguinness-okta) [inline _source/_docs/api/getting_started/releases-at-okta.md]: Do we need to define what "Stable" means?

File: _source/_docs/api/getting_started/releases-at-okta.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+## Early Access (EA)
+
+A feature in an Early Access (EA) release is new or enhanced functionality made available for customers to selectively "opt-in" to and use in both Production and non-Production environments. 
+Features in EA are supported by Okta Customer Support, SLAs, and announced in Okta's weekly Release Notes. 
+Any bugs or improvements will be managed and fixed with the same timeline and processes as those in General Availability.
+
+Features in EA release are marked with the EA icon: {% api_lifecycle ea %}
+
+## General Availability (GA)
```
Comment (karlmcguinness-okta) [inline _source/_docs/api/getting_started/releases-at-okta.md]: Similar comments to EA on that product features & APIs may have separate lifecycles and drift.  Some APIs will skip Beta and EA and go right to GA.  Some APIs are effectively GA but the features its depends on is EA so its gated on GA of dependencies.  Not suggesting we need to communicate all this complexity but I do think we need to make it clear that product features & APIs may have separate life cycles and what may be different for a feature vs an API for the lifecycle.

Comment (mystiberry-okta) [inline _source/_docs/api/getting_started/releases-at-okta.md]: These are all great comments, serious thank you! Will address them all in a new draft to include your comments and others that have been raised in person or email.

File: _source/_docs/api/getting_started/releases-at-okta.md (hunk 4 above)
Comment (mystiberry-okta) [inline _source/_docs/api/getting_started/releases-at-okta.md]: Cindy pointed out the unpleasant connotation that we would ever ship "unstable" features, so trying to reword this, all we're trying to say is that in beta, api may change without warning and is not backwards compatible with previous iterations in the beta.

File: _source/_docs/api/getting_started/releases-at-okta.md (hunk 2 above)
Comment (mystiberry-okta) [inline _source/_docs/api/getting_started/releases-at-okta.md]: FYI, @keithcasey-okta has asked for this level of specifity. I can definitely elevate/highlight the concept, will do.

File: _source/_docs/api/getting_started/releases-at-okta.md (hunk 1 above)
Comment (mystiberry-okta) [inline _source/_docs/api/getting_started/releases-at-okta.md]: I'm always dancing on the razor's edge of scaring people/not promising the wrong thing. Will work on it!

File: _source/_docs/api/getting_started/releases-at-okta.md (hunk 2 above)
Comment (keithcasey-okta) [inline _source/_docs/api/getting_started/releases-at-okta.md]: This is to delineate who is in charge of helping customers and setting expectations throughout the Beta process. I'd like to be more concise but anything shorter hasn't explained it effectively.

File: _source/_docs/api/getting_started/releases-at-okta.md (hunk 4 above)
Comment (keithcasey-okta) [inline _source/_docs/api/getting_started/releases-at-okta.md]: In this case, "stable" means "fixed" so maybe: backwards compatible, established, ?

File: _source/_docs/api/getting_started/releases-at-okta.md (hunk 1 above)
Comment (mystiberry-okta) [inline _source/_docs/api/getting_started/releases-at-okta.md]: Used the same language, pointed to that section, and made it a note.

File: _source/_docs/api/getting_started/releases-at-okta.md (hunk 3 above)
Comment (mystiberry-okta) [inline _source/_docs/api/getting_started/releases-at-okta.md]: Added.

Comment (mystiberry-okta) [inline _source/_docs/api/getting_started/releases-at-okta.md]: @karlmcguinness-okta Not sure how to translate this for customers:

- APIs might go from beta to GA if they have no product feature flag dependencies

-

File: _source/_docs/api/getting_started/releases-at-okta.md (hunk 4 above)
Comment (mystiberry-okta) [inline _source/_docs/api/getting_started/releases-at-okta.md]: Here's the new table row:

| API  Changes  | Subject to change | Subject to change | Backwards compatible  | Backwards compatible   | N/A   |

File: _source/_docs/api/getting_started/releases-at-okta.md (hunk 3 above)
Comment (karlmcguinness-okta) [inline _source/_docs/api/getting_started/releases-at-okta.md]: Key point is that not all APIs will go through the full lifecycle.  The lifecycle is dependent on whether its linked to a product features, risk of change, and degree in confidence that the API can be maintained in a backwards compatible way.  "Enhancements" often are released as GA and never have beta or EA while entire new APIs will usually have a beta or EA depending on whether its providing functionality for existing product features or new product features.

tl;dr;  folks should always expect a beta or EA

File: _source/_docs/api/getting_started/releases-at-okta.md (hunk 4 above)
Comment (karlmcguinness-okta) [inline _source/_docs/api/getting_started/releases-at-okta.md]: "Strict Versioning" maybe?
